openedx_external_enrollments/external_enrollments/salesforce_external_enrollment.py

```diff
@@ -14,6 +15,9 @@
 from openedx_external_enrollments.external_enrollments.base_external_enrollment import BaseExternalEnrollment
 from openedx_external_enrollments.models import ProgramSalesforceEnrollment
 
+CUSTOM_BUNDLE_TYPE = 'special offer'
```

Comment:
Since this is particular to the SalesforceEnrollment class, you can move it there, what do you think @Jacatove?
```suggestion
```